sharktank/sharktank/utils/testing.py

```diff
@@ -196,21 +197,25 @@ def assert_close_safetensors(
 
     print(f"Asserting tensors close: actual={actual_path}, ref={ref_path}")
 
-    not_close_list: list[tuple[str, str]] = []
+    ref_path = Path(ref_path)
+    actual_path = Path(actual_path)
 
-    if os.path.isdir(ref_path):
-        assert os.path.isdir(actual_path)
-        actual_path = os.path.abspath(actual_path)
-        ref_path = os.path.abspath(ref_path)
-        ref_paths = [
-            Path(root) / file
-            for root, dirs, files in os.walk(ref_path)
-            for file in files
+    if not ref_path.is_file():
```

Comment:
`ref_path.is_file()` will return file if the path doesn't exist. It's better to check that it exists and that it's a dir.

Reply:
I added a check if the path exists.